app/presenters/tree_builder.rb

```diff
@@ -172,7 +172,8 @@ def add_to_sandbox
         :leaf       => @options[:leaf],
         :add_root   => true,
         :open_nodes => [],
-        :lazy       => true
+        :lazy       => true,
+        :checkable  => false
```

Comment:
There is a high chance that this line can break a lot of things in other trees :(